newrelic_httprb.gemspec

```diff
@@ -0,0 +1,23 @@
+require File.expand_path('../lib/newrelic_httprb/version', __FILE__)
+
+Gem::Specification.new do |gem|
+  gem.authors       = ["Tiago Sousa"]
+  gem.email         = ["tiago.joao@gmail.com"]
+  gem.description   = %q{New Relic instrumentation for http.rb}
+  gem.summary       = %q{New Relic instrumentation for http.rb}
+  gem.homepage      = "https://github.com/Talkdesk/newrelic_httprb"
+  gem.license       = "MIT"
+
+  gem.name          = "newrelic_httprb"
+  gem.require_paths = ["lib"]
+  gem.version       = NewRelicHTTP::VERSION
+
+  gem.files         = `git ls-files`.split($/)
+  gem.test_files    = gem.files.grep(%r{^(test|spec|features)/})
+
+  gem.add_dependency 'newrelic_rpm', '~> 3.11'
```

Comment:
It's the minimum newrelic_rpm dependency that will work with latest versions of contrib gems we use:

[newrelic_moped](https://github.com/stevebartholomew/newrelic_moped/blob/master/newrelic_moped.gemspec#L16): `~> 3.11`
[padrino-rpm](https://github.com/Asquera/padrino-rpm/blob/master/padrino-rpm.gemspec): `~> 3.3.3`
[newrelic-redis](https://github.com/evanphx/newrelic-redis/blob/master/newrelic-redis.gemspec#L27): `~> 3.11`
[newrelic-zookeeper](https://github.com/jcmfernandes/newrelic-zookeeper/blob/master/newrelic-zookeeper.gemspec#L38): `~> 3.11`

This will also mean some systems (looking at you CTI) will have to update their NewRelic version.

Makes sense?